### PR TITLE
[JUJU-2472] Add --integrations flag on juju-status

### PR DESCRIPTION
**Pull request by nvinuesa**
When trying to "integrate" 2 times in a row, juju complains like this:
```
$ juju integrate lxd overlord:admin/cos.grafana-dashboards
ERROR cannot add relation "grafana-dashboards:grafana-dashboard lxd:grafana-dashboard"
relation grafana-dashboards:grafana-dashboard lxd:grafana-dashboard (already exists):

Use 'juju status --integrations' to view the current integrations.
```
This `--integrations` flag doesn't exist, so this PR adds it. 

_Note: we keep the `--relations` flag at least for a minor version as proposed by @jameinel._
## Checklist

*If an item is not applicable, use `~strikethrough~`.*

- [X] Code style: imports ordered, good names, simple structure, etc
- [ ] ~Comments saying why design decisions were made~
- [X] Go unit tests, with comments saying what you're testing
- [X] [Integration tests](https://github.com/juju/juju/tree/develop/tests), with comments saying what you're testing
- [ ] ~[doc.go](https://discourse.charmhub.io/t/readme-in-packages/451) added or updated in changed packages~

## QA steps

*Commands to run to verify that the change works.*

On a properly bootstrapped controller, with 2 applications that are integrated (for example following https://juju.is/docs/olm/get-started-with-juju and deploying PostgreSQL+Mattermost and integrate them)

```sh
juju status --integrations
```

Then the default (tabular) output should contain the integrations list at the end.

_Note: I also replaced the wording on the tabular output (for example, from the tests):_
```
Integration provider      Requirer                   Interface  Type         Message
mysql:juju-info           logging:info               juju-info  subordinate  
mysql:server              wordpress:db               mysql      regular      suspended  
wordpress:logging-dir     logging:logging-directory  logging    subordinate  
```
## Documentation changes

The `status` command now has a `--integrations` flag (we also keep `--relations`). Also, all the output formats contain the "Integration" wording (`Integration provider` in tabular, `integrations` in yaml and json).

## Bug reference

https://bugs.launchpad.net/juju/+bug/1999271

### CODE REVIEW

File: cmd/juju/status/status.go [hunk 1]
```diff
@@ -174,7 +174,8 @@ func (c *statusCommand) SetFlags(f *gnuflag.FlagSet) {
 
 	f.BoolVar(&c.color, "color", false, "Use ANSI color codes in tabular output")
 	f.BoolVar(&c.noColor, "no-color", false, "Disable ANSI color codes in tabular output")
-	f.BoolVar(&c.relations, "relations", false, "Show 'relations' section in tabular output")
```
Comment (barrettj12) [inline cmd/juju/status/status.go:177]: Do we need to add a note somewhere (visible to the user) that this flag is deprecated?

**Review by barrettj12 — APPROVED** [review]
Changes LGTM but I would suggest a more accurate title for this PR, e.g. "Change relation -> integration in `juju status`".

**Comment by barrettj12** [comment]
Also, should we target this to the 3.1 branch, or are we considering it a "bug" in 3.0?

**Comment by SimonRichardson** [comment]
@nvinuesa shall we close this, it's decaying and we shouldn't leave PRs open.

**Pull request closed without merging**